src/resources/localization/french_fr-fr.mcpl

```diff
@@ -840,7 +840,7 @@ Charger un Monde
 New World
 Creer un Monde
 
-Join Online World
```

Comment:
keep in mind this outdates all the translations.

Reply:
Translations should not be much different